pages/database-management/authentication-and-authorization/auth-system-integrations.mdx

```diff
@@ -9,42 +9,59 @@ import { Tabs, Tab } from 'nextra/components'
 
 # Auth system integrations (Enterprise)
 
-Memgraph supports authentication and (optional) authorization using a **custom
-built external auth module**. The two supported operation modes are:
-- [DEPRECATED] authentication only (username/password verification)
-- authentication and authorization (username/password verification and user to
-  role mapping)
-
-When a user connects to Memgraph, the database will forward the user's supplied
-username and password to the external auth module and wait for it to deliver
-the authentication and authorization verdict back to the database. Based on
-the returned verdict, Memgraph will either close the connection to the
-connected user or will allow the connection and associate the connection 
-with the appropriate role.
-
-When Memgraph is switched to use the external auth module for authentication
-its internal users are automatically disabled. All users are authenticated only
-using the module, existing local users are ignored. In order to authorize query 
-execution, the auth module must return a role associated with the user. This 
-role must be registered locally on Memgraph.
+Memgraph supports authentication and authorization using **external auth
+modules**. There’s built-in support for basic (username + password) auth via
+LDAP, and single-sign on over the SAML and OIDC protocols.
+
+When a user is connecting, Memgraph will forward the user’s supplied
+credentials (username + password, or scheme and auth response) to the external
+auth module and retrieve its verdict. Based on that, Memgraph will let the user
+connect, or in case of failure, close the connection.
 
 As this is an Enterprise feature, once the Memgraph Enterprise license expires,
 newly created users will be granted all privileges. The existing users'
 privileges will still apply but you won't be able to manage them.
 
-## Configuration flags
+## Configuration
+
+### Roles
+
+User roles must be defined in Memgraph before using auth modules. This is
+because auth modules return the role associated with the user. 
+
+### Flags
 
 Use the following configuration flags to configure the external auth module
 authentication and authorization mechanisms used by Memgraph.
 
  Flag                               | Description
 ------------------------------------|------------
- `--auth-module-executable`         | Path to the executable that should be used for user authentication/authorization.
+ `--auth-module-mappings`           | Associates auth schemes to external modules. A mapping is structured as follows: `<scheme>:<absolute path to module>` and individual entries are separated with `;`. If the mapping contains whitespace, enclose the flag value with quotation marks.
+ `--auth-module-executable`         | [DEPRECATED] Path to the executable that should be used for user authentication/authorization. Superseded by `--auth-module-mappings=basic` from Memgraph 2.18.
```

Comment:
It should also be marked as deprecated in https://memgraph.com/docs/database-management/configuration#auth-module

Reply:
Change `Superseded by` to `Replaced by`.

Reply:
Done